rfai/requests_for_ai_handler.py

```diff
@@ -1,10 +1,34 @@
+from common.logger import get_logger
+from common.repository import Repository
+from common.utils import Utils
+from rfai.config import NETWORK_ID, NETWORK, SLACK_HOOK
+from rfai.constant import REQUIRED_KEYS_FOR_GET_RFAI_EVENT
+from rfai.service.rfai_service import RFAIService
+
+rfai = RFAIService(repo=Repository(net_id=NETWORK_ID, NETWORKS=NETWORK))
+util = Utils()
+logger = get_logger(__name__)
+
+
 def request_handler(event, context):
     try:
-        # valid_event = validate_dict(
-        #     data_dict=event, required_keys=REQUIRED_KEYS_FOR_GET_RFAI_EVENT)
-        # if not valid_event:
-        #     return generate_lambda_response(400, "Bad Request", cors_enabled=True)
-        print(event)
-        # response = generate_lambda_response(200, {"status": "success", "data": ""}, cors_enabled=True)
+        logger.info(event)
+        valid_event = util.validate_dict(
+            data_dict=event, required_keys=REQUIRED_KEYS_FOR_GET_RFAI_EVENT)
+        if not valid_event:
```

Comment:
check how decorator works in python and see if we can use it here and put it common module